test/e2e/ibc_transfer_e2e_test.go

```diff
@@ -96,7 +96,7 @@ func (s *IBCTransferTestSuite) Test1IBCTransfer() {
 		}
 		return math.Abs(float64(balanceA.Sub(transferCoin).AmountOf(denom).Int64()-
 			balanceA2.AmountOf(denom).Int64())) < delta
-	}, 10*time.Second, 1*time.Second, "Transfer was not successful")
+	}, 1*time.Minute, 1*time.Second, "Transfer was not successful")
```

Comment:
why increase that much?

Reply:
Hmm probably IBC transfer packets take some time? This was for fixing the flaky e2e test AFAIK